travel/travel.py

```diff
@@ -25,6 +25,17 @@
 from .res_config import get_basic_passenger_limit
 
 
+def _get_travel_states(self, cr, uid, ids=None, context=None):
+    return [
+        ('draft', _('Draft')),
+        ('open', _('Saved')),
+        ('booking', _('In Reservation')),
+        ('reserved', _('Reserved')),
+        ('confirmed', _('Confirmed')),
+        ('done', _('Closed')),
+    ]
```

Comment:
You're going to need to regenerate the pot files because these terms are now code